sos/report/plugins/mlx5_core.py

```diff
@@ -22,6 +22,7 @@ class Mlx5_core(Plugin, IndependentPlugin):
     def setup(self):
         self.add_copy_spec([
             '/sys/kernel/debug/mlx5/0000:*/*',
+            '/sys/class/*/*/device/sriov_numvfs',
```

Comment:
need to collect it from the networking plugin
and the infiniband from infiniband plugin